### PR TITLE
Update README.md to include installation methods other than F-Droid

### LINKED ISSUE #5051: Update README.md to include other ways to install
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Update the README. It only lists F-Droid which is often late on releases (and today is late by months on releases due to a bug).

#### Is your feature request related to a problem? Please describe it

There are other ways to get NewPipe (our F-Droid repository, GitHub release page) and they should be listed, and explain how to choose one.

#### Additional context

See teamnewpipe/website#163

#### How will you/everyone benefit from this feature?

Users checking our README will know what are ways to install NewPipe.

### PR DESCRIPTION
**Pull request by mhmdanas**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [X] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- [X] Rename "updates" section to "installation"
- [x] Add a hidden span with ID `#updates` so as not to break old links
- [X] Update "installation" section to account for F-Droid bug
- [X] (Temporarily) remove the F-Droid badge

@B0pol suggested that we get somebody to design a NewPipe-y version of the [F-Droid badge](https://gitlab.com/fdroid/artwork/-/blob/master/badge). Until then, I think we should remove the badge altogether.

~~@B0pol what do you think I should do exactly?~~

#### Fixes the following issue(s)

-  Fixes #5051

#### APK testing 

Unnecessary.

#### Due diligence
- [X] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by kujaw** [comment]
Isn't installing NewPipe Legacy also one of temporary solutions? 
I've just installed it and it works. Additionally I don't have to install any repo or download apk from the website.

**Comment by Stypox** [comment]
@mhmdanas is this ready?

**Comment by mhmdanas** [comment]
@Stypox no. I totally forgot about this PR. I'll finish it tomorrow.

**Comment by mhmdanas** [comment]
Done. You can review now.

File: README.md [hunk 1]
```diff
@@ -80,14 +82,17 @@ NewPipe supports multiple services. Our [docs](https://teamnewpipe.github.io/doc
...
-## Updates
-When a change to the NewPipe code occurs (due to either adding features or bug fixing), eventually a release will occur. These are in the format x.xx.x . In order to get this new version, you can:
- 1. Build a debug APK yourself. This is the fastest way to get new features on your device, but is much more complicated, so we recommend using one of the other methods.
- 2. Add our custom repo to F-Droid and install it from there as soon as we publish a release. The instructions are here: https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
- 3. Download the APK from [Github Releases](https://github.com/TeamNewPipe/NewPipe/releases) and install it as soon as we publish a release.
- 4. Update via F-droid. This is the slowest method of getting updates, as F-Droid must recognize changes, build the APK itself, sign it, then push the update to users.
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+
+## Installation
```
Comment (Stypox) [inline README.md:88]: Maybe "Installation and updates"?

**Review by Stypox — CHANGES_REQUESTED** [review]
I like this, thank you for looking into it
<sub>When will F-Droid solve the issue?!?!?</sub>

File: README.md (hunk 1 above)
Comment (mhmdanas) [reply README.md]: Done.

**Review by Stypox — APPROVED** [review]

**Review by TobiGr — APPROVED** [review]

**Comment by TobiGr** [comment]
We need a list of README translators, so we can ping them when changing the English version and the changes can be applied to the other READMEs as fast as possible.

**Comment by Stypox** [comment]
Can't READMEs be pushed to Weblate?

**Pull request merged**